tests/test_main.py

```diff
@@ -136,11 +136,16 @@ def test_jp2pym_add_treated_as_replace():
 
 
 def test_jp2pym_raise_on_move():
-    patches = [{"op": "move", "path": "/name", "from": "/old_name"}]
+    patches = [{"op": "move", "path": "/name"}]
     with pytest.raises(JsonPatch2PyMongoException):
-        jsonpatch2pymongo(patches)
+        jsonpatch2pymongo(patches)    
```

Comment:
Please remove trailing white spaces :)

Reply:
Oh absolutely! Ran it through your standard `black -l 100` requirements.